demux/stheader.h

```diff
@@ -70,19 +71,19 @@ struct mp_codec_params {
     enum stream_type type;
 
     // E.g. "h264" (usually corresponds to AVCodecDescriptor.name)
-    const char *codec;
+    _Atomic(const char *) codec;
```

Comment:
i don't think the braces are needed?

Reply:
It's either this or `const char *_Atomic codec;`. Do you prefer this one?